src/server.cpp

```diff
@@ -1523,16 +1536,13 @@ void Server::SendBreath(u16 peer_id, u16 breath)
 	Send(&pkt);
 }
 
-void Server::SendAccessDenied(u16 peer_id, AccessDeniedCode reason, const std::string &custom_reason)
+void Server::SendAccessDenied(u16 peer_id, AccessDeniedCode reason,
+		const std::string &custom_reason, bool reconnect)
 {
 	DSTACK(__FUNCTION_NAME);
 
 	NetworkPacket pkt(TOCLIENT_ACCESS_DENIED, 1, peer_id);
-	pkt << (u8) reason;
-
-	if (reason == SERVER_ACCESSDENIED_CUSTOM_STRING) {
-		pkt << custom_reason;
-	}
+	pkt << (u8) reason << custom_reason << (u8)reconnect;
```

Comment:
You know, this is mostly a question of preference. I like it this way, you like it your way. I won't block you when you make it your way, why do you block me?